nautobot/core/tests/test_events.py

```diff
@@ -113,7 +140,78 @@ def test_publish_events_to_redis(self):
                 sub.get_message(timeout=5.0),
                 {"type": "pmessage", "pattern": "nautobot.*", "channel": "nautobot.test.event", "data": '{"b": 2}'},
             )
+
+            # Assert exclude topics are not published
+            publish_event(topic="nautobot.test.event.no-publish", payload={"c": 3})
+            self.assertNotEqual(
+                sub.get_message(timeout=5.0),
```

Comment:
Should `sub.get_message(timeout=5.0)` just return None in the case where there's no published message in that time? Might be better to just assert that instead?